client/src/Pages/Landing/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Container, Divider, Typography } from '@material-ui/core';
+
+import CardContainer from '../../Components/CardContainer';
+import useStyles from './style';
+
+function Landing() {
+  const classes = useStyles();
+
+  const [houses, setHouses] = useState([]);
+  const [errorMsg, setErrorMsg] = useState('');
+
+  const sortByTopRating = (housesData) => {
+    const cloningHouse = [...housesData];
+    return cloningHouse.sort((a, b) => {
+      if (a.rating > b.rating) {
+        return -1;
+      }
+      if (a.rating < b.rating) {
+        return 1;
+      }
+      return 0;
+    });
+  };
+
+  const sortByRecentlyAdded = (housesData) => {
+    const cloningHouse = [...housesData];
+    return cloningHouse.sort((a, b) => {
+      if (a.created_at > b.created_at) {
+        return -1;
+      }
+      if (a.created_at < b.created_at) {
+        return 1;
+      }
+      return 0;
+    });
+  };
+
+  async function fetchingData(source, limit = 6, skip = 0) {
+    try {
+      const { data } = await axios.get(
+        `/api/v1/houses?limit=${limit}&skip=${skip}`,
```

Comment:
* declare a function expression, instead of the function declaration
* you can send a request without sending a skip (it's by default 0) 
* and pass the value directly to the request, its not a dynamic values (you decide the number you didn't get from the user and will not change)
 ```
  async function fetchingData() {
    try {
      const { data } = await axios.get(
        '/api/v1/houses?limit=6',
```